src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/Checker.java

```diff
@@ -102,6 +102,7 @@
 import jdk.javadoc.internal.doclint.HtmlTag.AttrKind;
 import jdk.javadoc.internal.doclint.HtmlTag.ElemKind;
 import static jdk.javadoc.internal.doclint.Messages.Group.*;
+import jdk.javadoc.internal.doclets.toolkit.util.Utils;
```

Comment:
This (ugly) dependency would go away if you put that new method into the `Trees` API.